functions/go/set-namespace/main.go

```diff
@@ -1,75 +1,97 @@
+// This file will be processed and embedded to pluginator.
+
 package main
 
 import (
 	"fmt"
 	"os"
-	"strings"
 
+	"sigs.k8s.io/kustomize/api/k8sdeps/kunstruct"
+	"sigs.k8s.io/kustomize/api/resmap"
+	"sigs.k8s.io/kustomize/api/resource"
+	"sigs.k8s.io/kustomize/api/types"
 	"sigs.k8s.io/kustomize/kyaml/fn/framework"
-	"sigs.k8s.io/kustomize/kyaml/yaml"
+	"sigs.k8s.io/yaml"
 )
 
-var namespace string
-var allowClusterScoped bool
+//nolint
+func main() {
```

Comment:
What's the --help string for this command?

The usage message should a) provide a description of this function b) Describe the function config required.

Example:

https://github.com/GoogleContainerTools/kpt-functions-sdk/blob/master/ts/hello-world/src/label_namespace.ts#L29

Reply:
How was this addressed?

Reply:
`usage` function provides the --help message.